di_art/test/di_art_test.dart

```diff
@@ -1,13 +1,31 @@
+import 'package:di_art/error/errors.dart';
 import 'package:flutter_test/flutter_test.dart';
```

Comment:
Sorry, but shouldn't we use pure dart testing instead of flutter? Not sure if we need flutter dependencies in the core module.
Don't you think it's better to use this package: https://pub.dev/packages/test ? 😄

Reply:
dependencies were automatically added when creating the project. In addition, I observed errors when trying to remove flutter tests.

Reply:
Replaced `flutter_test` with pure dart `test` package without any issues locally. Maybe you could try to do that again? I think we should completely remove any flutter dependencies there, as we planned that core part could be used in a pure dart project, without flutter sdk in it :)